bezpy/mt/io.py

```diff
@@ -123,6 +123,9 @@ def read_xml(fname):
     site.Z = np.vstack([site.data['z_zxx'], site.data['z_zxy'],
                         site.data['z_zyx'], site.data['z_zyy']])
 
+    site.runlist = get_text(xml_site, "RunList").split()
```

Comment:
For all of the site properties you are updating, put `= None` in the site object. So, in the file `bezpy/mt/site.py` you would put:
```
site.runlist = None
site.runinfo = None
site.nimsID = None
```
Or any other properties you are adding to the object, so that we know they are updatable/available originally.

Reply:
Thanks